schemas/user/createUserSchema.js

```diff
@@ -26,7 +32,7 @@ module.exports = {
       },
     },
   },
-  lastNameNull: {
+  lastName: {
     custom: {
```

Comment:
change the custom field for  lastName: {
    notEmpty:{errorMessage:"lastName is null"}